common/djangoapps/util/testing.py

```diff
@@ -1,20 +1,19 @@
 import sys
 
+from mock import patch
+
 from django.conf import settings
 from django.core.urlresolvers import clear_url_caches, resolve
 
 
 class UrlResetMixin(object):
     """Mixin to reset urls.py before and after a test
-
```

Comment:
Note: I'm not sure where these whitespace changes came from. My intention was just to pick up @dan-f's changes-- I won't check these whitespace changes in.